src/components/SendQueryRequest.tsx

```diff
@@ -1,152 +0,0 @@
-import {
```

Comment:
Ripping out all of the old SamePage stuff to instead use the cross-app request protocol that is in development